cluster/kube-down.sh

```diff
@@ -27,6 +27,7 @@ if [ -f "${KUBE_ROOT}/cluster/env.sh" ]; then
 fi
 
 source "${KUBE_ROOT}/cluster/kube-util.sh"
+source "${KUBE_ROOT}/cluster/common.sh"
```

Comment:
Please don't add this here. These are designed to be sourced from cluster/${PROVIDER}/util.sh.